content/trips/cairngorms/ballater.md

```diff
@@ -1,15 +1,17 @@
 ---
-tags: cairngorms
-date: 2021-08-11
-title: Ballater
-kilometres: 29.6
-start: Braemar
-end: Candacraig
-average_speed: 16.7
-total_duration: "4:13:00"
 ascent: 190
+average_speed: 7.0
+date: 2021-08-11
 descent: 210
+end: Candacraig
+kilometres: 29
 permalink: false
+sidetrip_duration: 0:0:0
+sidetrip_km: 0
```

Comment:
should we count when we went into Braemar to see the local Highland games?

Reply:
added